### DISCUSSION TITLE
installation error

### DISCUSSION THREAD

**Discussion by haizhou0410** [mdolab/pygeo#228] [Q&A]
Hi, when I install pyGeo, the following error appeared. Could you give me some advice? Thank you!
checking for MPI compile and link ...
      /home/haizhou/anaconda3/bin/mpicc -pthread -B /home/haizhou/anaconda3/compiler_compat -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /home/haizhou/anaconda3/include -fPIC -O2 -isystem /home/haizhou/anaconda3/include -fPIC -I/home/haizhou/anaconda3/include/python3.11 -c _configtest.c -o _configtest.o
      /home/haizhou/anaconda3/bin/mpicc: line 330: x86_64-conda-linux-gnu-cc: command not found
      failure.
      removing: _configtest.c _configtest.o
      error: Cannot compile MPI programs. Check your configuration!!!
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for mpi4py
Successfully built pygeo
Failed to build mpi4py
ERROR: Could not build wheels for mpi4py, which is required to install pyproject.toml-based projects

**Comment by marcomangano** [comment]
It looks like you do not have either `MPI` or `mpi4py` installed, check the documentation for [MACH-Aero](https://github.com/mdolab/MACH-Aero/) package dependencies [here](https://mdolab-mach-aero.readthedocs-hosted.com/en/latest/installInstructions/install3rdPartyPackages.html)
